08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,38 @@
-/*Add CSS rules*/
\ No newline at end of file
+body {
+  font-family: sans-serif;
+  color: #555555;
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.parent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+section {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+article {
+  margin-right: 30px
+}
+
+img {
+  height: 230px;
+}
+
+button {
+  border: solid 1px;
+  background: transparent;
+  font-weight: bold;
+  color: #A1A6AD;
+  border-right: solid #8E1E02 4px;
+  padding: 10px;
+  width: 120px;
+  height: 40px;
```

Comment:
Make sure to change the cursor `cursor: pointer`. You want to do this if you are not wrapping the button in a `<a>' element. For example `<a href="#"><button>My Button</button></a>`